test/bigtest/tests/change-password-test.js

```diff
@@ -243,4 +243,56 @@ describeApplication('ChangePasswordPage', () => {
       expect(ChangePasswordPage.confirmPasswordField.errorMessage.isPresent).to.be.false;
     });
   });
+
+  describe('validation error for the incorrect new password, last ten passwords error', () => {
+    beforeEach(async () => {
+      const currentPassword = ChangePasswordPage.currentPasswordField;
+      const newPassword = ChangePasswordPage.newPasswordField;
+      const confirmPassword = ChangePasswordPage.confirmPasswordField;
+
+      await currentPassword.fillInput(lastTenPasswordsError);
+      await newPassword.fillInput('newPassword');
+      await confirmPassword.fillInput('newPassword');
+      await ChangePasswordPage.saveBtn();
+    });
+
+    it('all fields should not be reset upon failed submit', () => {
+      expect(ChangePasswordPage.currentPasswordField.value).to.equal(lastTenPasswordsError);
+      expect(ChangePasswordPage.newPasswordField.value).to.equal('newPassword');
+      expect(ChangePasswordPage.confirmPasswordField.value).to.equal('newPassword');
+    });
+
+    it('Validation error be present upon failed submit', () => {
```

Comment:
Write the description message in lowercase for consistency. Check other places.

Also, please check grammar.
```
validation error _should_ be ...
```